### PR TITLE
add  disable-compression arg for both operator and metrics-server

### PR DESCRIPTION
**Pull request by Adarsh-verma-14**
Signed-off-by: Adarsh-verma-14 [t_adarsh.verma@india.nec.com](mailto:t_adarsh.verma@india.nec.com)

Add  disable-compression arg for both operator and metrics-server.

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/charts/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))

### CODE REVIEW

**Comment by Adarsh-verma-14** [comment]
Hi @zroubalik , @JorTurFer , I have fixed CI checks failing of #326 . PTAL!!

File: keda/values.yaml [hunk 1]
```diff
@@ -44,6 +44,8 @@ operator:
   # You can run multiple replicas, but they will not improve the performance of KEDA, it could only reduce downtime during a failover.
   # Learn more in [our documentation](https://keda.sh/docs/latest/operate/cluster/#high-availability).
   replicaCount: 1
+  # --Disable response compression for k8s restAPI in client-go.
```
Comment (tomkerkhove) [inline keda/values.yaml:47]: That's a very technical thing to add; we may want to explain for non-Go experts/devs what it means

Comment (Adarsh-verma-14) [reply keda/values.yaml:47]: sure @tomkerkhove

Comment (zroubalik) [reply keda/values.yaml:47]: any update on this?

Comment (Adarsh-verma-14) [reply keda/values.yaml:47]: Hi @zroubalik , @tomkerkhove , I have added explanation for this.  PTAL!!

**Review by zroubalik — APPROVED** [review]

**Pull request merged**